ROLES.md

```diff
@@ -204,8 +214,14 @@ If a maintainer becomes inactive in the project for an extended period of time,
 emeritus maintainer. Emeritus maintainers lose their ability to approve code contributions, but retain their voting rights
 for up to one year. After one year, emeritus maintainers revert back to being normal members. 
 
-A maintainer becomes an emeritus maintainer if the individual hasn't contributed a PR to the project in 3 months. An emeritus 
-maintainer can regain full maintainer status by submitting 5 PRs to the project.
+Contributing to the project can be done in a number of ways:
```

Comment:
Should we be specific about contributing to the part of the codebase they are a maintainer for?

We could then remove the "maintained path" and "maintained repo" qualifiers below.

So something like:

Contributing to the maintained part of the codebase can be done in a number of ways:
- Merging pull requests
- Triaging issues or pull requests
...

Dunno if that is better or worse

Reply:
Done.